salt/modules/state.py

```diff
@@ -599,6 +598,22 @@ def apply_(mods=None,
         .. code-block:: bash
 
             salt '*' state.apply test localconfig=/path/to/minion.yml
+
+    sync
```

Comment:
Should this be something like `sync_mods`. The first thing this makes me think of is `synchronous` but maybe that's just me?

Reply:
I like that idea ^

Reply:
Yes, that does sound better, I will get that changed.